spec/api/made_to_measure/messages_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+describe 'MadeToMeasure::Messages' do
+  context 'v0.1' do
```

Comment:
Should we keep to this convention? I'm not sure I really like versioning apis especially at this early stage.